docs/PYPI.md

```diff
@@ -137,20 +128,33 @@ class Bot(Client):
     async def say(self, message: str):
         return message
 
+    @user_command
+    async def user_command(self, user: User):
+        return f"The user is {user}"
+
+    @message_command(name="Message command")
+    async def message_command(self, message: UserMessage):
+        return f"The message read \"{message.content}\""
```

Comment:
```suggestion
        return f'The message read "{message.content}"'
```

Reply:
I don't really like using '...' for quotes :skull: I'd rather flip them if we need to get rid of the escape character.

Reply:
Yeah, flip them then, not a fan of using single quotes either. I'm not sure but I think PEP8 prefers `'very "funny"'` over `"very \"funny\""`, at least that's what autopep8 for me (unless I totally messed up and it was a different formatter).